docs/how-to-guides/test-an-app.md

```diff
@@ -86,23 +92,6 @@ the previous test:
 }
 ```
 
-#### Generating input files automagically from existing topic data
```

Comment:
@MichaelDrogalis Why was this dropped? Can it be reinstated somewhere else? It's useful information.

Reply:
https://github.com/confluentinc/ksql/pull/6826